doc/api.rst

```diff
@@ -72,29 +72,43 @@ options as the constructor second argument::
 
 The following options are available:
 
-* ``debug``: When set to ``true``, the generated templates have a
+* ``debug`` *boolean*
+
+  When set to ``true``, the generated templates have a
   ``__toString()`` method that you can use to display the generated nodes
   (default to ``false``).
 
-* ``charset``: The charset used by the templates (default to ``utf-8``).
+* ``charset`` *string (default to ``utf-8``)*
+
+  The charset used by the templates.
+
+* ``base_template_class`` *(default to ``Twig_Template``)*
```

Comment:
Maybe state that is the charset of the `output`  as well?
(as this is something I know others find confusing (ie. think it is charset of the data you use in Twig))